src/app/components/Article/index.stories.jsx

```diff
@@ -1,5 +1,78 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
 import Article from './index';
+import { textBlock } from '../../models/blocks';
 
-storiesOf('Article', module).add('default', () => <Article />);
+const title = 'This is a title!';
+const lang = 'en-GB';
+
+const data = {
+  blocks: [
+    {
+      type: 'headline',
+      blockId: '1',
+      model: textBlock('This is a headline!'),
+    },
+    {
+      type: 'text',
```

Comment:
We should open an issue to address the fact that we can't reuse the `textBlock()` function for this model similar to how the headline model is being defined. We would need to handle the model logic similar to how we've handled the propTypes duplication. 

For this particular method we could have something like: 
```
export const textBlock = (text, id, paragraphs = 1) => ({
const paragraphModel = paragraphs.map(text, index) => {
  {
    type: 'paragraph',
    blockId: `p-${index}`,
    model: {
      text,
    },
  },
};

blocks: [
    {
      type: 'text',
      blockId: `t-${id}`,
      model: {
        blocks: [
          paragraphModel
        ],
      },
    },
  ],
});
```

This way it's extendable to multiple paragraphs and multiple uses of the same method in a single model

Reply:
There is already a ticket to add fixture data from a library, see https://github.com/bbc/simorgh/issues/243. Let's start from there and see if that is appropriate.